nucypher/utilities/clouddeploy.py

```diff
@@ -450,11 +534,11 @@ def _configure_provider_params(self, provider_profile):
 
         self._write_config()
 
-    def create_new_node_for_staker(self, address):
+    def create_new_node(self, node_name):
 
         response = requests.post("https://api.digitalocean.com/v2/droplets",
             {
-                "name": f'{self.namespace}-{address}',
+                "name": f'{node_name}',
                 "region": self.region,
                 "size": self.instance_size,
                 "image":"ubuntu-20-04-x64",
```

Comment:
is there a way to get the latest ubuntu version so we don't need to update this line every time? Or is this just the base that gets automatically updated to the latest?